### DISCUSSION TITLE
Idea: refactor the `Hifi` prefix into `H`

### DISCUSSION THREAD

**Discussion by PaulRBerg** [hifi-finance/hifi#80] [Ideas]
Proposed new contract name: `HPool` instead of `HifiPool`, `HPoolRegistry` instead of `HifiPoolRegistry`.

This would:

1. Reduce the net total size of our code base (slightly).
2. Make the nomenclature consistent with the H prefix in the HToken.

**Comment by scorpion9979** [comment]
I think it should be noted that this would have absolutely no effect on the compiled bytecode, especially since we have the compiler configured to ignore the metadata hash. Therefore, any decision we arrive at here should be entirely in favor of dev UX and readability.

Reference: https://ethereum.stackexchange.com/questions/114858/is-the-solidity-contract-class-name-of-any-relevance

**Reply by PaulRBerg** [reply]
Very good point. I am still in favor of this refactor, FWIW.
